src/java.base/share/classes/java/util/zip/ZipUtils.java

```diff
@@ -231,56 +221,56 @@ static final long GETSIG(byte[] b) {
+    static final int  ENDCOM(byte[] b) { return get16(b, 20);}  // size of ZIP file comment
+    static final int  ENDCOM(byte[] b, int off) { return get16(b, off + 20);}
 
     // zip64 end of central directory recoder fields
-    static final long ZIP64_ENDTOD(byte[] b) { return LL(b, 24);}  // total number of entries on disk
-    static final long ZIP64_ENDTOT(byte[] b) { return LL(b, 32);}  // total number of entries
-    static final long ZIP64_ENDSIZ(byte[] b) { return LL(b, 40);}  // central directory size
-    static final long ZIP64_ENDOFF(byte[] b) { return LL(b, 48);}  // central directory offset
-    static final long ZIP64_LOCOFF(byte[] b) { return LL(b, 8);}   // zip64 end offset
+    static final long ZIP64_ENDTOD(byte[] b) { return get64(b, 24);}  // total number of entries on disk
+    static final long ZIP64_ENDTOT(byte[] b) { return get64(b, 32);}  // total number of entries
+    static final long ZIP64_ENDSIZ(byte[] b) { return get64(b, 40);}  // central directory size
+    static final long ZIP64_ENDOFF(byte[] b) { return get64(b, 48);}  // central directory offset
+    static final long ZIP64_LOCOFF(byte[] b) { return get64(b, 8);}   // zip64 end offset
 
     // central directory header (CEN) fields
-    static final long CENSIG(byte[] b, int pos) { return LG(b, pos + 0); }
-    static final int  CENVEM(byte[] b, int pos) { return SH(b, pos + 4); }
-    static final int  CENVEM_FA(byte[] b, int pos) { return CH(b, pos + 5); } // file attribute compatibility
-    static final int  CENVER(byte[] b, int pos) { return SH(b, pos + 6); }
-    static final int  CENFLG(byte[] b, int pos) { return SH(b, pos + 8); }
-    static final int  CENHOW(byte[] b, int pos) { return SH(b, pos + 10);}
-    static final long CENTIM(byte[] b, int pos) { return LG(b, pos + 12);}
-    static final long CENCRC(byte[] b, int pos) { return LG(b, pos + 16);}
-    static final long CENSIZ(byte[] b, int pos) { return LG(b, pos + 20);}
-    static final long CENLEN(byte[] b, int pos) { return LG(b, pos + 24);}
-    static final int  CENNAM(byte[] b, int pos) { return SH(b, pos + 28);}
-    static final int  CENEXT(byte[] b, int pos) { return SH(b, pos + 30);}
-    static final int  CENCOM(byte[] b, int pos) { return SH(b, pos + 32);}
-    static final int  CENDSK(byte[] b, int pos) { return SH(b, pos + 34);}
-    static final int  CENATT(byte[] b, int pos) { return SH(b, pos + 36);}
-    static final long CENATX(byte[] b, int pos) { return LG(b, pos + 38);}
-    static final int  CENATX_PERMS(byte[] b, int pos) { return SH(b, pos + 40);} // posix permission data
-    static final long CENOFF(byte[] b, int pos) { return LG(b, pos + 42);}
+    static final long CENSIG(byte[] b, int pos) { return get32(b, pos + 0); }
+    static final int  CENVEM(byte[] b, int pos) { return get16(b, pos + 4); }
+    static final int  CENVEM_FA(byte[] b, int pos) { return Byte.toUnsignedInt(b[pos + 5]); } // file attribute compatibility
```

Comment:
Did you consider introducing `get8` for consistency here? As it stands, this looks like the odd one out.

Reply:
I considered it, but since `get8` would basically just delegate to or do exactly what `Byte.toUnsignedInt` does I opted to cut out the middle man.